### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by qw123wh** [qw123wh/new-clock#5]
@IzzySoft hi friend, i need your help again. I created a new, more updated version of this app https://github.com/qw123wh/Aospdeskclock
Can you please help me again to publish it on F-Droid?
The permissions are the same as the other app, maybe there are some more but they are all necessary
I created a screenshot folder

**Comment by IzzySoft** [comment]
Is it a new app, or rather a new version? In the first case, just follow the same procedure [as last time](https://github.com/qw123wh/Aospdeskclock/issues/1) (including setting up Fastlane structures). If you [open an RFP](https://gitlab.com/fdroid/rfp/issues), anyone at F-Droid can pick up from there. Or you could take a look at my MR from last time and open your own directly.

PS: I'd also suggest to choose a more distinguished name. You know how many clock apps out there are simply called "clock"? Further I'd stand back from superlatives ("best there is"), that sounds a bit exaggerating. No offence meant, but whenever I see such things in an app description, that very much tempts me to immediately skip over to something "more modest" :wink:

As last time, I've added your app [to my repo](https://apt.izzysoft.de/fdroid/index/apk/com.best.deskclock), where it will show up with the next sync tomorrow. For the remaining steps, I've gotta send you over to the mentioned places; I currently lack the time to fully process everything, sorry.

**Comment by IzzySoft** [comment]
I see you've opened an RFP, great! May I suggest an update to your `full_description.txt`?

```markdown
<i>New Clock</i> is an updated version of the AOSP desk clock, including alarms, clock, timer and stopwatch. Features include a.o.

* power off alarm (alarm will ring with phone off, snapdragon phones only, tested on all custom rom)
* flip and shake action for dismiss/postpone alarm
* possibility to turn off/postpone the alarm with the power button and volume button
* swipe to delete alarm
* Light and Dark mode
```

That's what's currently live in my repo. If possible, please also adapt this for the Italian description :wink:

Btw, for `short_description.txt` I've set: "alarm, clock, timer and stopwatch". Of course up to you if you want to adopt that.

Note: If you want your screenshots to appear in a certain order, prefix the file names with numbers (e.g. `01_Alarm.jpg`). If you want to do that, better do it before your app gets build by F-Droid (there's a long-standing bug with old screenshots not being removed – so once they're in, you can't get them out anymore; this does not apply to my repo, luckily).

**Comment by qw123wh** [comment]
I updated full_description.txt and short_description.txt, then i also renamed the screenshots as you suggested.
Let me know if I need to change other things, and thanks again for your invaluable help

**Comment by IzzySoft** [comment]
Gladly – and that looks great now, thanks! Do you plan any new release soon? Else we'll need to take care to specify the currently last commit with the build recipe (or the fastlane update would be ignored; fastlane is always used from the same commit the app is build from).

**Comment by qw123wh** [comment]
No new releases are planned for now.  If I manage to implement new features, I will do it only after the app is published on fdroid.

**Comment by IzzySoft** [comment]
OK, thanks! Could you please leave a comment on your RFP then what commit should be used, so it's not forgotten?

**Comment by qw123wh** [comment]
I left the comment on the RFP, I hope it goes well, I don't understand the question of the last commit

**Comment by IzzySoft** [comment]
See over there. I've prepared the metadata as best as I could. I might have gone for the merge request myself, but as you placed the `src` dir directly in the repo root, I'm unsure how to handle that (our bot was unsure as well). Hopefully a more experienced team member will take care for the rest soon.

**Comment by qw123wh** [comment]
So I have to make the structure similar to the other project then: app / scr / main / com / ....  ?
I can try to do it tomorrow, hoping not to do any damage

**Comment by IzzySoft** [comment]
I don't know if you *have* to – but if it were in that structure, I'd know how to deal with it. Otherwise we'd just wait for someone who knows, and I can learn from it. Either way is fine :smile:

**Comment by qw123wh** [comment]
I tried but if I change the app structure, Android Studio won't let me compile the app anymore. It gives me this error: "Android Manifest does not exist". I'm sorry, I hope someone can solve it.

**Comment by IzzySoft** [comment]
Let's see if one of our developers picks up the RFP and gives some advice there. As I'm no dev, I cannot help with this part.

**Comment by qw123wh** [comment]
i converted the project so it should be simpler i hope
https://github.com/qw123wh/new-clock-fdroid/issues/1
